src/QuadForm/ClosestVector.jl

```diff
@@ -268,24 +268,24 @@ function posQuadTriple_intVectors(QT::Array{Any,1}, E::Array{Any,1})
 end
   
 @doc Markdown.doc"""
-    closest_vectors(Q::MatrixElem, L::MatrixElem, c::RingElement) 
+    closest_vectors(Q::MatrixElem, L::MatrixElem, c::RingElement; bool=false) 
                                                     -> Array{Array{fmpz, 1}, 1}
   
   
-  Returns all the integer vectors of length n such that the inhomogeneous 
-  quadratic function corresponding to an n variabled quadratic triple is 
-  less than or equal to zero.
+  Returns all the integer vectors `x` of length n such that the inhomogeneous 
+  quadratic function `q_{QT}(x) := xQx + 2xL + c <= 0` corresponding to an n variabled
+  quadratic triple. If the optional argument bool=true, it returns
+  all vectors `x` such that `q_{QT}(x) = 0`. By default bool=false.
 """
-function closest_vectors(G::MatrixElem, L::MatrixElem, c::RingElement) 
+function closest_vectors(G::MatrixElem, L::MatrixElem, c::RingElement; bool=false) 
```

Comment:
Please rename `bool` to something more descriptive - for instance `equal::bool=false`:.